src/pharmpy/tools/amd/run.py

```diff
@@ -465,7 +465,7 @@ def _run_structsearch_tmdd(model):
             highest_ranked = min(rank_dict, key=rank_dict.get)
             extra_model = retrieve_models(path / 'modelsearch', names=[highest_ranked])[0]
         else:
-            extra_model = None
```

Comment:
I wanted `extra_model`to be None if there is no model with one less compartment. But if the model is None then the structsearch tool will raise an error because it does not accept input models that are None. Any suggestions here?